integration/314_container_accept_before_kretprobe_test.sh

```diff
@@ -0,0 +1,37 @@
+#! /bin/bash
+
+# shellcheck disable=SC1091
+. ./config.sh
+
+start_suite "Test accept before kretprobe, see https://github.com/weaveworks/tcptracer-bpf/issues/10"
+
+weave_on "$HOST1" launch
+
+# Launch the server before Scope
```

Comment:
Please elaborate why (e.g. _to make sure it calls accept() and the socket is in LISTEN mode before the eBPF kprobes are installed_).